cosmos/models/Workflow.py

```diff
@@ -552,7 +552,7 @@ def _run(workflow, session, task_queue):
 
         # only commit Task changes after processing a batch of finished ones
         session.commit()
-        time.sleep(.3)
+        time.sleep(5)
```

Comment:
I'm okay with the increase in drm_ge.py but less so with this one.  I usually test the example workflows using a local drm, and this could really slow that down

Do you think they're both necessary?